docs/source/guides/logical_types_and_semantic_tags.ipynb

```diff
@@ -0,0 +1,629 @@
+    "\n",
+    "For an in-depth guide on how to set and manipulate these types, see the [understanding types and tags guide](https://woodwork.alteryx.com/en/stable/guides/understanding_types_and_tags.html).\n",
+    "\n",
+    "For information on how to customize Woodwork's type system, see the [custom types and inference guide](https://woodwork.alteryx.com/en/stable/guides/custom_types_and_type_inference.html).\n",
+    "\n",
+    "It's important to remember that Woodwork columns will always have a logical type and that any semantic tags that are added by Woodwork are meant to add additional meaning onto the logical type. We'll start out by looking in-depth at semantic tags so that when we get to logical types, we can better understand how a semantic tag might add additional information onto it."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "czech-strip",
+   "metadata": {},
+   "source": [
+    "\n",
+    "# Semantic Tags\n",
+    "\n",
+    "Here is the full set of Woodwork-defined semantic tags:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "supposed-cookie",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import woodwork as ww\n",
+    "ww.list_semantic_tags()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "extra-husband",
+   "metadata": {},
+   "source": [
+    "#### Standard Tags\n",
+    "Standard tags are associated with specific logical types. They are useful for indicating predefined categories that logical types might fall into.\n",
+    "- `'numeric'` - Is applied to any numeric logical type\n",
+    "    - **Uses**: Can select for just numeric columns when performing operations that require numeric operations\n",
+    "    - **Related Properties**: `series.ww.is_numeric`\n",
+    "- `'category'` - Is applied to any logical type that is categorical in nature\n",
+    "    - **Uses**: Can select for just categorical columns when performing operations that require categories\n",
+    "    - **Related Properties**: `series.ww.is_categorical`\n",
+    "\n",
+    "#### Index Tags\n",
+    "Index tags are added by Woodwork to a DataFrame when an `index` or `time_index` column is identified by the user. These tags have some special properties that are only confirmed to be true in the context of a DataFrame (so any Series with these tags may not have these properties).\n",
+    "- `'index'` - Indicates that a column is the DataFrame's index, or primary key\n",
+    "    - There will only be one index column\n",
+    "    - The contents of an index column will be unique\n",
+    "    - An index column will have any standard semantic tags associated with its logical type removed\n",
+    "    - The data in an index column will be reflected in the DataFrame's underlying index\n",
+    "- `'time_index'`\n",
+    "    - There will only be one time index column\n",
+    "    - A time index column will contain either datetime or numeric data\n",
+    "\n",
```

Comment:
These lines are not being broken up into separate lines in the rendered document